apps/condo/domains/billing/schema/BillingReceiptFile.js

```diff
@@ -170,12 +193,18 @@ const BillingReceiptFile = new GQLListSchema('BillingReceiptFile', {
                 })
             }
         },
-        validateInput: async ({ resolvedData, addValidationError, existingItem }) => {
+        validateInput: async ({ resolvedData, addValidationError, existingItem, operation, context }) => {
+            // in case if receipt was resolved by model hook - let's skip validation
+            if (resolveReceiptByImportId(operation, get(resolvedData, ['receipt']), get(resolvedData, ['importId']))) {
```

Comment:
1) it’s not clear
2) it loves like a main logic branch. What If someone in a future will add some validation

Reply:
Going to put this validation inside if condition - just to keep simple it for understanding and possible for enrich it with other validations